client/results.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="results.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div id = "headerDiv">
+            <img src="https://cdn.vox-cdn.com/thumbor/Pkmq1nm3skO0-j693JTMd7RL0Zk=/0x0:2012x1341/1200x800/filters:focal(0x0:2012x1341)/cdn.vox-cdn.com/uploads/chorus_image/image/47070706/google2.0.0.jpg" alt="google logo">
+            <form id="form"> 
+<<<<<<< HEAD
+                <input type="search" id="query" name="q" placeholder="Search...">
+=======
+                <input type="search" id="query" name="q" placeholder="Search..." >
+>>>>>>> 47dcc907bfa2159d6ee7cfdc1549a79f002dc1cc
+                <button>Search</button>
+              </form>
+
+        </div>
+    </header>
+    <section>
+        <ul class="no-bullets"></ul>
+<<<<<<< HEAD
+            <li><a href="#">Item 1</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 2</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 3</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 4</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 5</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 6</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 7</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 8</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 9</a></li>
+            <p class="description">Description...............</p>
+            <li><a href="#">Item 10</a></li>
+            <p class="description">Description...............</p>
```

Comment:
could use a loop to produce these in javascript, and fetch links & data from server.